java/kotlin-extractor2/src/main/kotlin/entities/MethodCall.kt

```diff
@@ -32,7 +33,11 @@ fun KotlinFileExtractor.extractMethodCall(
     val args = call.valueArguments
         .map { arg ->
             val expr = arg.getArgumentExpression()
-            val p = argMapping[expr]
+
+            // `argMapping` seems to drill into parenthesized expressions
```

Comment:
Do you know where this happens?

Reply:
I don't, but I found the utility function `KtPsiUtil.deparenthesize` which does something similar but also handles `KtAnnotatedExpression` and `KtLabeledExpression`. So we might also need to cover these too.

Reply:
Let's add a TODO comment so we ensure that we handle all cases properly